views/layout.pug

```diff
@@ -22,5 +22,3 @@ html
     div.footer-nav
 
     include footer
-    script(src="/javascripts/index.js")
```

Comment:
Was this change intentional?  Wanted to verify since I'm not sure why we're removing sripts from the layout when working on the API.